MapboxCoreNavigation/DistanceFormatter.swift

```diff
@@ -68,21 +68,25 @@ struct RoundingTable {
         func measurement(for distance: CLLocationDistance) -> Measurement<UnitLength> {
             switch unit {
             case .millimeter:
-                return Measurement(value: distance * 1_000, unit: .millimeters)
+                return Measurement(value: distance, unit: .meters).converted(to: .millimeters)
```

Comment:
Good call, that’s a lot more elegant than what we had before. But if we stick to what we had before, then we can round after multiplying but before converting to a Measurement.